asb-ballerina/Ballerina.toml

```diff
@@ -4,7 +4,7 @@ name = "asb"
 version = "0.1.4-SNAPSHOT"
 license= ["Apache-2.0"]
 authors = ["Ballerina"]
-keywords = ["Azure","ServiceBus"]
+keywords = ["Azure","Service Bus","Messaging","Message Brokers"]
```

Comment:
Don't we need to start with a simple letter? in each keyword

Reply:
Resolved and amended the changes. Thank you for the suggestion.